common/company_types/99_basic_companies.txt

```diff
@@ -701,7 +704,8 @@ company_basic_shipyards = {
 	}
 
 	prosperity_modifier = {
-		unit_navy_offense_mult = 0.1
+		building_logging_camp_throughput_add = 0.1
+		interest_group_ig_industrialist_approval_add = 2
```

Comment:
This feels immersion breaking, why would a shipyard increase lumber throughput? =/

Reply:
My understanding of this from other companies that do similar things are that the companies sometimes increase the throughput of their primary source materials in order to simulate the beneficial supply lines of those goods. Sometimes they also just give unrelated bonuses, like Mitsui giving +10% iron mine throughput even though their goods are textiles, fertilizer, and lead.

Also, I've checked modifiers a few times and noticed that some don't show up even though it's all spelled right so I'm not sure what happened there.